### ISSUE TITLE
Make compatible with apollo-codegen

### ISSUE THREAD

**Issue by drewproud** [zth/graphql-generate-flow-schema-assets#1]
We use [apollo-codegen](https://github.com/apollographql/apollo-codegen) for generating our schema.json files, which saves the json file in a slightly different form than this library expects.

I'm getting the following error:

```js
const types = schema.__schema.types;
TypeError: Cannot read property 'types' of undefined
```

It expects the file in the form

```json
{
  "__schema": {
    ...
  }
}
```

but apollo-codegen outputs it in the form

```json
{
  "data": {
    "__schema": {
      ...
    }
  }
}
```

We can work around this for sure, but this is a super handy library and that would definitely be a nice-to-have. 

Thanks for putting this out there!

**Comment by zth** [comment]
Ahh, nice catch! Coming to think of it, I think there are a few other tools as well out there that outputs the schema in that format. I'll push a fix tomorrow night and, thanks for letting me know!

**Comment by zth** [comment]
This should be fixed in 0.9.2. Would you care to verify for me? Thanks!

**Comment by drewproud** [comment]
Works like a charm! Thanks a ton.
